### PR TITLE
Add registration form (closing `[F] Registration form #8`)

### PR DESCRIPTION
**Pull request by Karkavin**
All forms of authorization and registration are completed. I think it would be a good idea to create a new issue within which to write a field validator (on FE side). For example: e-mail validation.

### CODE REVIEW

File: angio-frontend/src/modules/user/models/user.ts [hunk 1]
```diff
@@ -68,3 +68,7 @@ export interface UserResetAccountModel {
     resetCode: string;
     newPassword: string;
 }
+
+export interface UserRegisterModel {
+    email: string;
+}
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/models/user.ts]: see: https://github.com/nikon-petr/angio/pull/18#discussion_r331665924

File: angio-frontend/tests/e2e/fixtures/modules/user/registration.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (nikon-petr) [inline angio-frontend/tests/e2e/fixtures/modules/user/registration.json]: see: https://github.com/nikon-petr/angio/pull/18#discussion_r331677673

File: angio-frontend/src/modules/user/views/Registration.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        </v-flex>
+    </CentredLayout>
+</template>
+
+<script lang="ts">
+    import {Component, Vue} from 'vue-property-decorator';
+    import CentredLayout from '@/modules/common/components/CentredLayout.vue';
+    import {throttle} from 'helpful-decorators';
+    import RegistrationForm from '@/modules/user/components/RegistrationForm.vue';
+    import {UserRegisterModel} from "@/modules/user/models/user";
```
Comment (nikon-petr) [inline angio-frontend/src/modules/user/views/Registration.vue]: Use single quotes instead of double quotes

**Comment by nikon-petr** [comment]
Dont't forget to link the pr to corresponding issue.

File: angio-frontend/src/modules/user/views/Registration.vue (hunk 3 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/views/Registration.vue]: Done

File: angio-frontend/tests/e2e/fixtures/modules/user/registration.json (hunk 2 above)
Comment (Karkavin) [reply angio-frontend/tests/e2e/fixtures/modules/user/registration.json]: Done

File: angio-frontend/src/modules/user/models/user.ts (hunk 1 above)
Comment (Karkavin) [reply angio-frontend/src/modules/user/models/user.ts]: Done

**Pull request merged**